Packs/HPE_Service_Manager/Integrations/HPE_Service_Manager/HPE_Service_Manager.yml

```diff
@@ -1248,4 +1282,32 @@ script:
       description: Device status.
     - contextPath: HPSM.Devices.UpdatedBy
       description: The last user who updated the device.
+    description: Find and return a device by ID.
+  - name: hpsm-create-request
+    arguments:
+    - name: title
+      required: true
+      description: The title of the request.
+    - name: purpose
+      required: true
+      description: The purpose of the request.
+    - name: callbackcontactname
```

Comment:
change to callBackContactName

Reply:
Unfortunately I do not have access to the system any more :-(